tst/testbugfix/2018-06-11-mapping.tst

```diff
@@ -0,0 +1,13 @@
+#
+# There was a bug where composition of an identify mapping and another mapping x where the Source of the identity mapping contained the
```

Comment:
identify -> identity

Also, could you please wrap your lines after 80 or so characters?